utils/notifications.js

```diff
@@ -875,6 +889,15 @@ const sendInstantNotification = async (requestData) => {
         },
       },
     ],
+    tracking_settings: {
+      subscription_tracking: {
+          enable: true,
+          html:
+            requestData.preferredLanguage.toString() !== fieldToConceptIdMapping.spanish.toString()
```

Comment:
This won't work for language checking. requestData.preferredLanguage has string value "english" or "spanish".

Reply:
Was this tested?

Reply:
I can not test it since it is called from biospecimen site.  Could you show me the value of ` requestData.preferredLanguage`?

Reply:
You should be able to host your Connect Faas locally and also run a local instance of Biospecimen Dashboard that points to your local API instance.

Reply:
solved and tested on biospecimen locally